src/PlusDataCollection/WinProbe/vtkPlusWinProbeVideoSource.cxx

```diff
@@ -19,6 +19,79 @@ See License.txt for details.
 //----------------------------------------------------------------------------
 vtkStandardNewMacro(vtkPlusWinProbeVideoSource);
 
+  //----------------------------------------------------------------------------
+  // Define command strings
+const char* vtkPlusWinProbeVideoSource::SET_FREEZE                   = "SetFrozen";
+const char* vtkPlusWinProbeVideoSource::GET_FREEZE                   = "GetFrozen";
+const char* vtkPlusWinProbeVideoSource::SET_TGC                      = "SetTGC";
+const char* vtkPlusWinProbeVideoSource::GET_TGC                      = "GetTGC";
+const char* vtkPlusWinProbeVideoSource::SET_ALL_FOCAL_DEPTHS         = "SetAllFocalDepths";
+const char* vtkPlusWinProbeVideoSource::SET_FOCAL_DEPTH              = "SetFocalDepth";
+const char* vtkPlusWinProbeVideoSource::GET_FOCAL_DEPTH              = "GetFocalDepth";
+const char* vtkPlusWinProbeVideoSource::SET_ALL_ARFI_FOCAL_DEPTHS    = "SetAllARFIFocalDepths";
+const char* vtkPlusWinProbeVideoSource::SET_ARFI_FOCAL_DEPTH        = "SetARFIFocalDepth";
+const char* vtkPlusWinProbeVideoSource::GET_ARFI_FOCAL_DEPTH        = "GetARFIFocalDepth";
+const char* vtkPlusWinProbeVideoSource::SET_B_MULTIFOCAL_ZONE_COUNT  = "SetBMultifocalZoneCount";
```

Comment:
In case you were trying to align the equal signs here, looks like your off by a space. No big deal and doesn't need a rebuild/retest, but in case you want to clean it up you can.